iset.mm

```diff
@@ -136322,6 +136322,21 @@ S C_ ( P ( ball ` D ) T ) ) $=
       ABVCVGBHTVJVFVMTVMBVGCVIVHVQVRUQVFVMVAURUSUTVB $.
   $}
 
+  ${
+    $d x A $.  $d x B $.  $d x F $.
+    cnlimci.f $e |- ( ph -> F e. ( A -cn-> D ) ) $.
+    cnlimci.c $e |- ( ph -> B e. A ) $.
+    $( If ` F ` is a continuous function, then the limit of the function at any
+       point equals its value.  (Contributed by Mario Carneiro,
+       28-Dec-2016.) $)
+    cnlimci $p |- ( ph -> ( F ` B ) e. ( F limCC B ) ) $=
```

Comment:
Shouldn't this be named according to other deduction style theorems, ending with a d?

Reply:
If there is a corresponding inference or closed form theorem, sure. But there isn't a rule that all deduction style theorems end in `d`, all implication style theorems end in `i`, all closed form theorem end in `t`, all theorems with a not-free hypothesis end in `f`, etc.

I don't know if you are looking a the `i` at the end, but the `i` suffix here means "implication" ("implication (suffix)" entry at https://us.metamath.org/mpeuni/conventions-labels.html ). https://us.metamath.org/mpeuni/cnlimc.html is a biconditional and https://us.metamath.org/mpeuni/cnlimci.html is one direction of that biconditional.